documentation/src/main/asciidoc/userguide/chapters/envers/Envers.adoc

```diff
@@ -1028,7 +1028,12 @@ This basic query allows retrieving entity names and corresponding Java classes c
 ====
 [source, JAVA, indent=0]
 ----
-include::{sourcedir}/EntityTypeChangeAuditTest.java[tags=envers-tracking-modified-entities-queries-example]
+include::{sourcedir}/EntityTypeChangeAuditTest.java[tags=envers-tracking-modified-entities-queries-example1]
+----
+
+[source, JAVA, indent=0]
+----
+include::{sourcedir}/EntityTypeChangeAuditTest.java[tags=envers-tracking-modified-entities-queries-example2]
```

Comment:
How is it related to this issue?

This commit is quite hard to follow :/

Reply:
The test fails now due to the change as previously we allowed entities having the same JPA name but with a different  class name.